t/unit/backends/test_cache.py

```diff
@@ -143,7 +143,7 @@ def test_as_uri_multiple_servers(self):
         assert b.as_uri() == backend
 
     def test_regression_worker_startup_info(self):
-        pytest.importorskip('memcached')
+        pytest.importorskip('memcache')
```

Comment:
```suggestion
        pytest.importorskip('memcached')
```

Reply:
isn't it memcached? https://memcached.org/

Reply:
no, the to-be-imported python module is named "memcache", see my comment below